src/openforms/forms/tests/factories.py

```diff
@@ -35,3 +36,17 @@ class Meta:
 
     form = factory.SubFactory(FormFactory)
     form_definition = factory.SubFactory(FormDefinitionFactory)
+
+
+class FormVersionFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = FormVersion
+
+    form = factory.SubFactory(FormFactory)
+    export_blob = {}
+
+    @factory.post_generation
+    def post(obj, create, extracted, **kwargs):
+        json_form = form_to_json(obj.form.id)
+        obj.export_blob = json_form
+        obj.save()
```

Comment:
Please follow the django model structure:

1. fields
2. Meta
3. methods

```suggestion
class FormVersionFactory(factory.django.DjangoModelFactory):
    form = factory.SubFactory(FormFactory)
    export_blob = {}

    class Meta:
        model = FormVersion

    @factory.post_generation
    def post(obj, create, extracted, **kwargs):
        json_form = form_to_json(obj.form.id)
        obj.export_blob = json_form
        obj.save()
```

Reply:
I was trying to be consistent with the factories defined above. Should I change the other factories too?

Reply:
Feel free to do so!